docs/install.rst

```diff
@@ -1,8 +1,12 @@
 Installation
 ============
 
-Install from pypi.org as you would any other Python module using your preferred package manager:
+Install from `pypi.org <https://pypi.org/project/cyclonedx-python-lib/>`
```

Comment:
tried to do as described here: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links

Reply:
Should be like this @jkowalleck:

```
Install from `pypi.org`_.

.. _pypi.org: https://pypi.org/project/cyclonedx-python-lib/
```